packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```diff
@@ -1142,9 +1142,6 @@
       "properties": {
         "defaultStatus": {
           "oneOf": [
-            {
```

Comment:
In vscode with a `oneOf`, if none of the sub-schemas match, vscode will show the error message from the first one, which we want to be the better message about enum values rather than the one about the value not being `null`.